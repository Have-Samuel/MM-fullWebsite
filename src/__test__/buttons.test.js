import renderer from 'react-test-renderer';
import Button from '../components/buttons/buttons';

it('render correctly', () => {
  const handleButtonClick = (btnContent) => btnContent;
  const tree = renderer
    .create(<Button handleButtonClick={handleButtonClick} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
