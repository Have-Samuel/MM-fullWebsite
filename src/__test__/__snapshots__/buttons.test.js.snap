// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render  1`] = `
<button
  className="keyboardBtn"
  id=""
  type="button"
/>
`;

exports[`render correctly 1`] = `
<button
  className="keyboardBtn"
  id=""
  type="button"
/>
`;
